docs/content/en/docs/tasks/cluster/cluster-delete.md

```diff
@@ -74,5 +77,5 @@ GitOps field not specified, clean up git repo skipped
 🎉 Cluster deleted!
 ```
 
-This will delete all of the VMs that were created in your provider.
+For vSphere, this will delete all of the VMs that were created in your provider.
```

Comment:
We should add a note saying for bare metal the servers will be powered off if the BMC information had been provided.